.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,20 +25,13 @@ Check the box below to confirm that you have in fact seen this (put an X in the
 
 **Changelog**
 <!--
-Here you can fill out a changelog that will automatically be added to the game when your PR is merged.
-
-Only put changes that are visible and important to the player on the changelog.
-
-Don't consider the entry type suffix (e.g. add) to be "part" of the sentence:
-bad: - add: a new tool for engineers
-good: - add: added a new tool for engineers
-
-Putting a name after the :cl: symbol will change the name that shows in the changelog (otherwise it takes your GitHub username)
-Like so: :cl: PJB
--->
+Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
 
 :cl:
 - add: Added fun!
 - remove: Removed fun!
 - tweak: Changed fun!
 - fix: Fixed fun!
```

Comment:
What I'd do is make it a separated comment block, so people don't need to uncomment out the info part above